pkg/controller/storageclusterinitialization/noobaa_system_reconciler.go

```diff
@@ -57,8 +60,16 @@ func (r *ReconcileStorageClusterInitialization) newNooBaaSystem(initialData *ocs
 		},
 
 		Spec: nbv1.NooBaaSpec{
-			StorageClassName: &storageClassName,
-			CoreResources:    &coreResources,
+			DBStorageClass:            &storageClassName,
+			PVPoolDefaultStorageClass: &storageClassName,
+			CoreResources:             &coreResources,
+			DBResources:               &dbResources,
+			Tolerations:               defaults.DaemonPlacements["noobaa-core"].Tolerations,
+			DBVolumeResources: &corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: resource.MustParse("50Gi"),
```

Comment:
sound this `50Gi` volume be something that we have a tunable exposed for?

Reply:
@davidvossel Ultimately this volume should auto-grow, but RBD doesn't support resizing volumes yet right?
In any case the initial size should be defined somewhere - it can be added as an OCS spec property, but it will still have a default hard-coded somewhere, right?

Reply:
@davidvossel, I will add it to `DaemonResources` in defaults, so it will be easier to change later. 
I agree with @guymguym, we will have to decide if we want to add a property in the spec for it but I am not sure it's required now